### PR TITLE
Prepare to rename default branch

### PR DESCRIPTION
**Pull request by rco-ableton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,5 +99,5 @@ Aqt StyleSheets is distributed under the MIT license (see LICENSE).
 
 ## CI Status
 
-Master: [![Build status](https://ci.appveyor.com/api/projects/status/6uv86juxwv7nlu3e/branch/master?svg=true)](https://ci.appveyor.com/project/gck-ableton/aqt-stylesheets/branch/master)
-[![Build Status](https://travis-ci.com/Ableton/aqt-stylesheets.svg?branch=master)](https://travis-ci.com/Ableton/aqt-stylesheets)
+Main: [![Build status](https://ci.appveyor.com/api/projects/status/6uv86juxwv7nlu3e/branch/main?svg=true)](https://ci.appveyor.com/project/gck-ableton/aqt-stylesheets/branch/main)
```
Comment (mst-ableton) [inline README.md:102]: Will this work? Won't we have to wait for AppVeyor to build on `main` first, before we can get the URL?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 branches:
   only:
-    - master
+    - main
```
Comment (ala-ableton) [inline .travis.yml:3]: We don't use Travis CI anymore, so this file can be deleted.

File: README.md [hunk 3]
```diff
@@ -99,5 +99,5 @@ Aqt StyleSheets is distributed under the MIT license (see LICENSE).
...
+[![Build Status](https://travis-ci.com/Ableton/aqt-stylesheets.svg?branch=main)](https://travis-ci.com/Ableton/aqt-stylesheets)
```
Comment (ala-ableton) [inline README.md:103]: We don't use Travis CI anymore, so this line can be removed.

File: README.md (hunk 1 above)
Comment (ala-ableton) [reply README.md:102]: @rco-ableton You're correct.

**Review by mst-ableton — APPROVED** [review]
lgtm!

**Pull request merged**